### PR TITLE
update ios release notes v1.2.3

### PR DESCRIPTION
**Pull request by ChenLev**

### CODE REVIEW

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ navigation_weight: 0
 ---
 
 # Release Notes
+## 1.2.3 - 2019-12-17
 
+### Fixed.
+- Calling `conversation.getEvents` returned an `NXMEvent` with the field `fromMember` set to `nil`
+- Description in errors.
```
Comment (mheap) [inline _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md]: @ChenLev Can you add more information here? What do you mean by `Description in errors`?

**Comment by pardel** [comment]
checks fail from forks so added https://github.com/Nexmo/nexmo-developer/pull/2286 instead

File: _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md (hunk 1 above)
Comment (pardel) [reply _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md]: @mheap changed that in https://github.com/Nexmo/nexmo-developer/pull/2286/files

Comment (ChenLev) [reply _documentation/en/client-sdk/sdk-documentation/ios/release-notes.md]: @mheap i don't have permission to edit,
is this better?
`- `NSError` object with domain:`NXMErrorDomain` has now error description in `error.userInfo[NSLocalizedDescriptionKey]`.`

**Pull request closed without merging**